SparkPost/2web2ui
# FE-858 - Cypress tests for the Alerts list and details pages

[FE-858](https://jira.int.messagesystems.com/browse/FE-858)

### What Changed

- Updates Cypress NPM scripts to use the [Cypress module API](https://docs.cypress.io/guides/guides/module-api.html) instead of the CLI. This allows for a distinct (themed!) configuration of test batches
- Adds new tests for the details and list page for alerts:
  * `/alerts/details/<id>`
  * `/alerts`
- Adds `data-id` attributes to various components


### How To Test
- Verify tests are passing
- Visit the aforementioned app routes and verify that tests properly cover feature functionality and requirements (including edge cases)

### To Do
